install: cannot stat 'browserpass': No such file or directory

```
% wget https://github.com/browserpass/browserpass-native/releases/download/3.0.1/browserpass-linux64.zip
% unzip browserpass-linux64.zip

% DESTDIR=/home/sergio/.local PREFIX= make install 
install -Dm755 -t "/home/sergio/.local/bin/" browserpass
install: cannot stat 'browserpass': No such file or directory
make: *** [Makefile:80: install] Error 1

% cp browserpass-linux64 browserpass         
% DESTDIR=/home/sergio/.local PREFIX= make install
install -Dm755 -t "/home/sergio/.local/bin/" browserpass
install -Dm644 -t "/home/sergio/.local/lib/browserpass/" Makefile
install -Dm644 -t "/home/sergio/.local/share/licenses/browserpass/" LICENSE
install -Dm644 -t "/home/sergio/.local/share/doc/browserpass/" README.md
install -Dm644 browser-files/chromium-host.json   "/home/sergio/.local/lib/browserpass/hosts/chromium/com.github.browserpass.native.json"
install -Dm644 browser-files/chromium-policy.json "/home/sergio/.local/lib/browserpass/policies/chromium/com.github.browserpass.native.json"
install -Dm644 browser-files/firefox-host.json    "/home/sergio/.local/lib/browserpass/hosts/firefox/com.github.browserpass.native.json"
```

--- a/README.md
+++ b/README.md
@@ -26,8 +26,8 @@ This is a host application for [browserpass](https://github.com/browserpass/brow
 
 The following operating systems provide a browserpass package that can be installed using a package manager:
 
--   TODO
--   TODO
+-   [Arch Linux](https://aur.archlinux.org/packages/browserpass/)
+-   [NixOS](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/security/browserpass/default.nix) - also read [Install on Nix / NixOS](#install-on-nix--nixos)
 
 Once the package is installed, **refer to the section [Configure browsers](#configure-browsers)**.
 
@@ -52,7 +52,21 @@ Primary key fingerprint: EB4F 9E5A 60D3 2232 BB52  150C 12C8 7A28 FEAC 6B20
 
 Unpack the archive. If you decided to compile the application yourself, refer to the [Building the app](#building-the-app) section on how to do so. Once complete, continue with the steps below.
 
-Configure the hosts json files using `make configure` and then install the app using `sudo make install` (if you compiled it using `make browserpass`) or `sudo make BIN=browserpass-XXXX install` (if you downloaded a release with pre-built binary). Both `configure` and `install` targets respect `PREFIX` and `DESTDIR` parameters if you want to customize the install location (e.g. to avoid `sudo`).
+If you downloaded a release archive with pre-compiled binary, follow these steps to install the app:
+
+```
+make BIN=browserpass-XXXX configure      # Configure the hosts json files, XXXX refers to archive name, e.g. "linux64"
+sudo make BIN=browserpass-XXXX install   # Install the app, XXXX refers to archive name, e.g. "linux64"
+```
+
+If you compiled the app yourself, you can omit `BIN` parameter:
+
+```
+make configure      # Configure the hosts json files
+sudo make install   # Install the app
+```
+
+In addition, both `configure` and `install` targets respect `PREFIX`, `DESTDIR` parameters if you want to customize the install location (e.g. to install to a `$HOME` dir to avoid using `sudo`).
 
 Finally proceed to the [Configure browsers](#configure-browsers) section.
 

